setup.sh

```diff
@@ -95,35 +118,21 @@ browser.comp-pat[jules_doc.x] = (?msx-i:\A // [^/]+ /svn/ ([^/]+) /*(.*) \z)" >>
 
 ## Install JULES
```

Comment:
**Last step missing: Caching your MOSRS password**

#Caching your MOSRS password may be problematic if your MOSRS password has special characters in it (eg "%","!" or "&")

#Download two MOSRS utilities from [here]# (https://code.metoffice.gov.uk/trac/home/wiki/AuthenticationCaching/GpgAgent):

   #firefox https://code.metoffice.gov.uk/trac/home/raw-attachment/wiki/AuthenticationCaching/GpgAgent/mosrs-setup-gpg-agent

#Put in your MOSRS password, save the file and close Firefox.

   #firefox https://code.metoffice.gov.uk/trac/home/raw-attachment/wiki/AuthenticationCaching/GpgAgent/mosrs-cache-password

#Again, put in your MOSRS password, save the file and close Firefox.

```
 mv ~/Downloads/mosrs-setup-gpg-agent ~/.local/mosrs-setup-gpg-agent

   mv ~/Downloads/mosrs-cache-password ~/.local/mosrs-cache-password

   export PATH=$HOME/.local:$PATH

   chmod -R 755 ~/.local
```

#Now activate
`   . mosrs-setup-gpg-agent`

#To test this has worked correctly
`rosie hello`
#svn info --username=[yourusername] https://code.metoffice.gov.uk/svn/test

#you will need to re-cache your MOSRS password every time you log in from now on.
`export PATH=$HOME/.local:$PATH`
`[[ "$-" != *i* ]] && return`
`. mosrs-setup-gpg-agent`